src/main/java/org/candlepin/subscriptions/tally/InventoryAccountUsageCollector.java

```diff
@@ -371,10 +371,7 @@ public static Host hostFromHbiFacts(
   @Transactional
   @Timed("swatch_hbi_system_reconcile")
   public void reconcileSystemDataWithHbi(String orgId, Set<String> applicableProducts) {
-    if (!accountServiceInventoryRepository.existsById(
-        AccountServiceInventoryId.builder().orgId(orgId).serviceType(HBI_INSTANCE_TYPE).build())) {
-      accountServiceInventoryRepository.save(new AccountServiceInventory(orgId, HBI_INSTANCE_TYPE));
-    }
+    accountServiceInventoryRepository.saveIfDoesNotExist(orgId, HBI_INSTANCE_TYPE);
```

Comment:
This PR does more than allow tally by org synchronously. Please update the PR title and description to include a little more detail about what this line in particular addresses. I found it a little confusing when looking at the PR as a whole.

Reply:
Updated.